src/tools/admin/conseillers/desactiver.js

```diff
@@ -0,0 +1,78 @@
+
+const { program } = require('commander');
+const { Pool } = require('pg');
+
+const { execute } = require('../../utils');
+
+const pool = new Pool();
+
+execute(__filename, async ({ db, logger, exit }) => {
+  const getConseiller = async id => {
+    console.log('id:', typeof id);
+    try {
+      const { rows } = await pool.query(`
+        SELECT
+          id,
+          disponible
+        FROM djapp_coach
+        WHERE id = $1`,
+      [id]);
+      console.log('rows:', rows);
+      return rows;
+    } catch (error) {
+      logger.info(`Erreur DB : ${error.message}`);
+    }
+  };
+
+  const updateConseiller = async (id, disponibleChange) => {
+    try {
+      const { rows } = await pool.query(`
+        UPDATE djapp_coach
+        SET
+          disponible
+        WHERE id = $1`,
+      [id, disponibleChange]);
+      return rows;
+    } catch (error) {
+      logger.info(`Erreur DB : ${error.message}`);
+    }
+  };
+
+  program.option('-d, --disponible <true/false>', 'disponible: désactivé le conseiller par la valeur false ');
```

Comment:
désactiver

Reply:
Y'aurait la possibilité de gérer avec les options --disponible et --no-disponible (sans avoir à spécifier le paramètre "true" ou "false"), mais je laisse @loicguillois voir.

https://github.com/tj/commander.js/#other-option-types-negatable-boolean-and-booleanvalue